static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1679,3 +1679,64 @@ $quote-mark: 35px;
         background-color: $garnett-neutral-1;
     }
 }
+
+////// **Crossword******************************** ////
+.crossword__container {
+    .crossword__clues-header {
+        border-top: 1px solid $lifestyle-garnett-main-1;
+    }
+}
+
+.crossword__controls {
+    .button--primary {
+        background: $lifestyle-garnett-feature;
+        border: 1px solid $lifestyle-garnett-feature;
+        &:hover,
+        &:visited,
+        &:focus {
+            background-color: darken($lifestyle-garnett-feature, 10%);
+            border-color: darken($lifestyle-garnett-feature, 10%);
+        }
+    }
+
+    .button--secondary {
+        background: $garnett-neutral-8;
+        border: 1px solid $garnett-neutral-8;
+        &:hover,
+        &:visited,
+        &:focus {
+            background-color: darken($garnett-neutral-8, 10%);
+            border-color: darken($garnett-neutral-8, 10%);
+        }
+    }
+
+    .button--secondary.crossword__controls__button--confirm {
+        background-color: $garnett-neutral-1;
+        color: $garnett-neutral-5;
+        border: 0;
+    }
+}
+
+.crossword__anagram-helper-outer {
+    .button--large {
+        background: $lifestyle-garnett-feature;
+        border: 1px solid $lifestyle-garnett-feature;
+        &:hover,
+        &:visited,
+        &:focus {
+            background-color: darken($lifestyle-garnett-feature, 10%);
+            border-color: darken($lifestyle-garnett-feature, 10%);
+        }
+    }
+
+    .button--tertiary {
+        background: transparent;
+        border: 1px solid $garnett-neutral-4;
+        &:hover,
+        &:visited,
+        &:focus {
+            background-color: transparent;
+            border-color: darken($garnett-neutral-4, 15%);
+        }
+    }
+}
```

Comment:
Maybe these styles belong somewhere in the [`crosswords`](https://github.com/guardian/frontend/tree/master/static/src/stylesheets/module/crosswords) folder. What do you think?

Reply:
@SiAdcock I have moved styles to scss in the crosswords folder.